napari/utils/colormaps/colormaps.py

```diff
@@ -111,6 +117,31 @@ def _low_discrepancy_image(image, seed=0.5, margin=1 / 256):
     return image_out
 
 
+def color_dict_to_colormap(color_dict):
+    """
+    Generate a color map based on the given color dictionary
+    Parameters
+    ----------
+    color_dict : dict of label and color string matches
+
+    Returns
+    -------
+    colormap : Colormap with provided control colors
+    custom_color : dict of label to color control point within colormap
+    """
+    custom_color = {0: 0.0}
+    colors = ['#000000'] + [
```

Comment:
Naive question but why doesn't `'#000000'` here mean the background gets mapped to black instead of transparent? That is the hex code for black isn't it?